kahuna/public/js/components/gr-collections-panel/gr-collections-panel.js

```diff
@@ -37,13 +39,55 @@ grCollectionsPanel.controller('GrCollectionsPanelCtrl', [
     ctrl.selectedCollections = selectedCollections;
 }]);
 
+grCollectionsPanel.factory('collectionsTreeState', ['$window', function($window) {
+    // TODO: Add garbage collection to state.
+    const localStorageKey = 'collectionsTreeOpen';
+    const jsonStr = $window.localStorage.getItem(localStorageKey) || '[]';
+
+    // A little bit of superstition in case this was set weirdly before.
+    let jsonArr = [];
+    try {
+        let jsonArr = JSON.parse(jsonStr);
+        if (!Array.isArray(jsonArr)) {
+            throw new Error(`Invalid ${localStorageKey} json: ${jsonArray}`);
+        }
```

Comment:
Sorry I misread what you said, and thus missed my error.
Removed the unnecessary error and just set the value.
I've also updated the `localStorage` value to stop it happening again.